torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp

```diff
@@ -251,7 +255,7 @@ void unpackQuantizedWeightsHelper(
           ser_tup->elements()[0].isString()) {
         auto elements = ser_tup->elements();
         auto version = elements[0].toStringRef();
-        TORCH_INTERNAL_ASSERT(version == "2", "Unknown serialization version");
+        TORCH_INTERNAL_ASSERT(version == "3", "Unknown serialization version");
```

Comment:
I don't think we want to change this.  Shouldn't be necessary if you don't change the v2 serializer.